SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,85 @@
+    +------------------------+--------------+-----------------------+----------------------+
+
+
+.. table::
+
+    +------------------+---------------+-------------------------+-----------------------+
+    | **Visibility**   | **Operation** | **Tensor**              | **Number of Parties** |
+    +==================+===============+=========================+=======================+
+    | Public & Private | Addition      | - ReplicatedShareTensor | - 3                   |
+    |                  |               | - ShareTensor           | - 2                   |
+    |                  +---------------+-------------------------+-----------------------+
+    |                  | Subtraction   | - ReplicatedShareTensor | - 2+                  |
+    |                  |               | - ShareTensor           | - 2                   |
+    +------------------+---------------+-------------------------+-----------------------+
+
+
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Mult, matmul, conv2d, conv2d_transpose  | - Falcon     | - 3                   |
+    |                  |                                         | - SPDZ       | - 2+                  |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Equal to                                | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Non Equal to                            | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Greater than                            | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Greater or Equal than                   | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less than                               | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less or Equal than                      | - FSS        |                     2 |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | XOR                                     | - FSS        | - 2                   |
```

Comment:
Falcon supports XOR for 3 parties, but only the public. Maybe in Protocol, you can mention "Falcon (public only)"